swap_meet/item.py

```diff
@@ -1,2 +1,26 @@
+import uuid
 class Item:
-    pass
\ No newline at end of file
+    
+    #Instantiate with id, allowing custom or using UUID Int auto generate 
+    def __init__(self, id=None, condition=None):
+        self.id = id if id is not None else uuid.uuid1().int
+        self.condition = condition if condition is not None else 0
+    #get the class name 
+    def get_category(self):
+        return str(type(self).__name__)
+
+    #here I am overriding the __str__ method. 
+    def __str__(self):
+        return (f"An object of type {self.get_category()} with id {self.id}")
+
+    #adding a condition description class to describe condition based on value of condition
+    def condition_description(self):
```

Comment:
clever, but unnecessarily so here.  A `dict` would have sufficed, and been O(1) lookup, vs O(n) to fall through n conditions in the worse case :)